### PR TITLE
Make RISparser into a two-way project

### PR DESCRIPTION
**Pull request by J535D165**
This PR implements the following:

- More consistent API for reading RIS files. `readris` is now `load` (see json, pickle,... )
- Deprecates `readris`.
- Add functionality to dump (write) RIS files with the function `dump`
- Rename the package to fit the purpose and remove uppercase characters (PEP8 https://www.python.org/dev/peps/pep-0008/#package-and-module-names)

Looking forward to your feedback.

### CODE REVIEW

**Comment by J535D165** [comment]
Solves #7

**Comment by J535D165** [comment]
Any remarks? @MrTango

**Comment by MrTango** [comment]
It would awesome oif someone could integrate the test into Github actions or travisCI.

**Comment by shapiromatron** [comment]
this looks fantastic, thanks for such a large contribution!  I can help shepherd this later to completion if you need it later this week. We should bump the version to 0.5.0 for this change and document the breaking changes.

**Comment by shapiromatron** [comment]
I can do github actions @MrTango later this week in a separate PR; I know I mentioned it before. Any objects to black style linting enforcement?

**Comment by MrTango** [comment]
> I can do github actions @MrTango later this week in a separate PR; I know I mentioned it before. Any objects to black style linting enforcement?

no, i use it too in new projects, go ahead ;)

File: README.rst [hunk 1]
```diff
@@ -1,17 +1,21 @@
-Pure Python RIS files parser
-============================
+Pure Python RIS files parser and reader
+=======================================
+
+Before, this project was named 'RISparser'.
```
Comment (shapiromatron) [inline README.rst]: move this link to `HISTORY.rst`; indicate the api renaming for breaking changes

File: README.rst [hunk 2]
```diff
@@ -20,6 +24,16 @@ Usage
...
 
+Writing:
+
+.. code:: python
+
+   >>> import os
+   >>> import rispy
+   >>> filepath = 'tests/example_export_full.ris'
+   >>> with open(filepath, 'w') as bibliography_file:
+   ...     rispy.dump(entries, bibliography_file)
```
Comment (shapiromatron) [inline README.rst]: could you create a simple `entries` object in the docs to demonstrate the minimal requirements?

File: rispy/writer.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            self._get_reference_type(ref)
+        ))
+
+        for label, value in ref.items():
+
+            # not available
+            try:
+                tag = self._rev_mapping[label.lower()]
+            except KeyError:
+                logging.info("label {} not exported".format(label))
```
Comment (shapiromatron) [inline rispy/writer.py]: change to warning?

File: rispy/writer.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+def dump(references, file, mapping=None):
+    """Dump a ris lines to file.
+
+    Entries are codified as dictionaries whose keys are the
+    different tags. For single line and singly occurring tags,
+    the content is codified as a string. In the case of multiline
+    or multiple key occurrences, the content is returned as a list
+    of strings.
+
+    Keyword arguments:
```
Comment (shapiromatron) [inline rispy/writer.py]: revise format slightly here to more closely follow google style comments: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

It looks like you followed the prior convention, @MrTango is it ok if we progressively switch to a standard comment style?

File: setup.py [hunk 5]
```diff
@@ -21,7 +21,7 @@ def find_version():
         raise RuntimeError('Unable to find version string.')
 
 setup(
-    name='RISparser',
+    name='rispy',
```
Comment (shapiromatron) [inline setup.py]: i like this change.

**Comment by J535D165** [comment]
Let me know if there is anything to do for me in this.

File: README.rst (hunk 1 above)
Comment (J535D165) [reply README.rst]: Not completely sure what to do with this one. Added the changelog to HISTORY.rst.

**Comment by J535D165** [comment]
Thanks for your review. I made some changes to resolve the remarks.

**Comment by MrTango** [comment]
maybe it should be better 1.0 ;)

**Comment by J535D165** [comment]
Sounds valid. 

> maybe it should be better 1.0 ;)

Sounds valid. I will leave the preparation for the new release up to you.

**Comment by shapiromatron** [comment]
Looks great, thanks for the contribution! I'll work on minting a 1.0 release later this week (per discussion above); this will be the big new feature!

**Pull request merged**